compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -270,12 +259,14 @@ struct QuantizeActivation final : public luci::CircleNodeMutableVisitor<bool>
 
 struct QuantizeBias final : public luci::CircleNodeMutableVisitor<bool>
 {
-  QuantizeBias(loco::DataType input, loco::DataType output) : input_type(input), output_type(output)
+  QuantizeBias(loco::DataType input, loco::DataType output, QuantizationGranularity granularity)
+      : input_type(input), output_type(output), granularity(granularity)
```

Comment:
`granularity(granularity)` is same variable. does this work correctly?

Reply:
when I execute this code, it works properly. However, I can change it if you think it is confused.

Reply:
Ah.. thanks, let's change it like `granularity(gr)`  :)

Reply:
AFAIK, It works well

Reply:
AFAIK, it works well:)

Reply:
This indeed works well, but we usually use underscore prefix for private variables, e.g. "_granularity".
`QuantizeBias` should be a class rather than a struct, and the variables should be private.
It's just a note, please proceed without my approval if others find changes OK.